airflow/providers/tableau/hooks/tableau.py

```diff
@@ -133,3 +139,37 @@ def get_all(self, resource_name: str) -> Pager:
         except AttributeError:
             raise ValueError(f"Resource name {resource_name} is not found.")
         return Pager(resource.get)
+
+    def get_job_status(self, job_id: str) -> Enum:
```

Comment:
This return type is almost useless; use `TableauJobFinishCode` instead.

Similarly, the `rtype` below is not useful to anyone; since the type is not useful on its own, I would use prose to explain it with `:return:` instead (something like *An `Enum` that describe the Tableau job’s return code*).

Reply:
Thank you for your suggestion. With `TableauJobFinishCode` in docs it is more clear. I changed it also in `wait_for_state` function.